### ISSUE TITLE
textBox to support input type password.

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#556]
### Proposed behavior
Have an explicit API for writing text on the password field

### Actual behavior
Today `write(“password”,into(textBox({id:"password_box_id"})))` does not work as `inputBox` API only look for `//input[@type="text"]` where as the given input box is of `//input[@type="password"]` type. One has to use the `inputField` API to write on to password field box.

Its not very obvious that password fields are treated differently. Considering that it is a common use-case, a separate API will help.

### Notes
Maybe have an API like `passwordBox(<selector details>)`

**Comment by saikrishna321** [comment]
@sguptatw IMHO we should just allow write API to send text into any inputField

**Comment by saikrishna321** [comment]
@NivedhaSenthil @zabil Ping

**Comment by zabil** [comment]
> we should just allow write API to send text into any inputField

Agree. We must change the current API to handle this.

**Comment by saikrishna321** [comment]
@sguptatw can you please provide an example for this?

**Comment by NivedhaSenthil** [comment]
inputField is so generic to use as it can have many [types](https://www.w3schools.com/tags/att_input_type.asp) will be better if anything looking like a textBox is handled by `textBox` api and create other apis for respective types like we have `radiobutton` `checkbox` etc. We can deprecate `inputField` in that case.

Example for password field:
 https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_password 
currently `textBox` api can't be used for writing into password as @sguptatw has mentioned.

**Comment by sguptatw** [comment]
@saikrishna321  Example for the above.
```
const { openBrowser, goto, below, textBox, into, write, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("https://s1.demo.opensourcecms.com/wordpress/wp-admin/plugins.php");
        await write("opensourcecms",into(textBox(below("Username or Email Address"))));
        await write("opensourcecms",into(textBox({id:"user_pass"})));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```

**Comment by NivedhaSenthil** [comment]
inputField is so generic to use as it can have many [types](https://www.w3schools.com/tags/att_input_type.asp) will be better if anything looking like a textBox is handled by `textBox` api,  like we have separate apis for radiobutton and checkbox any other type that looks different has to have different api and inputField should not be used. We can deprecate `inputField` in that case.

Example for password field:
 https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_password 
currently `textBox` api can't be used for writing into password as @sguptatw has mentioned.

**Comment by NivedhaSenthil** [comment]
Fix available in 50181a9. `inputField` is now deprecated and `textBox` can be used to fetch password fields.

**Comment by Debashis9012** [comment]
Now we can use textBox api for password type field. 
Based on the observation this enhancements has been fixed with the commit version: ```50181a9```
